tensorflow/core/kernels/depthwise_conv_grad_op.cc

```diff
@@ -570,8 +570,16 @@ class DepthwiseConv2dNativeBackpropInputOp : public OpKernel {
     // For in_depth == 1 and grouped convolutions.
     use_cudnn_ = CanUseCudnn() && std::is_same<Device, GPUDevice>::value;
     cudnn_use_autotune_ = CudnnUseAutotune();
-    use_cudnn_grouped_conv_ = false;
     dtype_ = DataTypeToEnum<T>::value;
+#if CUDNN_VERSION >= 7603
```

Comment:
Should this be a regular `if (CUDNN_VERSION >= 7603)` so that both branches of the `if` keep building?